src/style.css

```diff
@@ -1,3 +1,79 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+html {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
 }
+
+img {
+  max-width: 100%;
+  display: block;
+}
+
+.header {
+  margin: 0 auto;
+  max-width: 1200px;
+  min-width: 1024px;
+  height: 60px;
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  list-style: none;
+  margin-left: 20px;
+  height: 60px;
+  line-height: 60px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: inline-block;
+  position: relative;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  height: 60px;
+  line-height: 60px;
+  cursor: pointer;
```

Comment:
add transition to make your changes smooth on hover

Reply:
cursor: pointer it is by default for a tag